### PR TITLE
Msvs extend sources

### PR DESCRIPTION
**Pull request by mortenvp**

### CODE REVIEW

**Comment by mortenvp** [comment]
Can be seen in action here: https://github.com/steinwurf/function_ref/blob/initial/wscript

**Comment by petya2164** [comment]
@mortenvp I think this is a clever solution!
So we just have to make sure that all wscripts call the extend function in the affected repos.

File: project_generators/msvs.py [hunk 1]
```diff
@@ -767,6 +770,11 @@ def collect_source(self):
 
         # Try to include all existing files in the project
         self.source = self.ctx.srcnode.ant_glob('**', excl=exclude_files)
+
+        for srcnode in self.msvs_extend_sources:
+            print("Extend sources: {}".format(srcnode.ant_glob('**', excl=exclude_files)))
```
Comment (petya2164) [inline project_generators/msvs.py]: Maybe we should kill this print. Converting an ant_glob to string prints all the items, right?
Now imagine that we extend sources for a massive project ;)

Comment (mortenvp) [reply project_generators/msvs.py]: Ah yes, the print should be removed - will do that

**Comment by petya2164** [comment]
Maybe I should test this magic on the function_ref project before merging this.

**Comment by mortenvp** [comment]
Yes, please test it

**Comment by petya2164** [comment]
It works well on function_ref. It added all the source files that I care about.

**Comment by mortenvp** [comment]
:+1: then I guess we can merge and release?

**Comment by petya2164** [comment]
Yes, this will do the job and it only requires a little bit of extra work in each affected project. So I am reasonably content with this solution.

**Pull request merged**